tests/wpt/metadata/html/browsers/the-window-object/window-open-noopener.html.ini

```diff
@@ -25,3 +25,9 @@
   [noopener window.open targeting _self]
     expected: NOTRUN
 
+  [noopener window.open targeting _parent]
+    expected: NOTRUN
+
+  [noopener window.open targeting _top]
+    expected: NOTRUN
+
```

Comment:
This is again a loop creating tests, which previously didn't get past the first iteration. Failure is due to `BroadcastChannel` being undefined...